CONTRIBUTING.md

```diff
@@ -61,3 +61,7 @@ are the good start.
 
 We use [git-backup-actions](https://github.com/jenkins-zh/git-backup-actions/) to backup this repo into 
 [gitee](https://gitee.com/jenkins-zh/jenkins-cli).
+
+## go-bindata
+
+If you use MacOS and you don't have go-bindata, you can use command `brew install go-bindata` to install it
```

Comment:
I think you might not get my point here. We already have something about this in the `Makefile`.People who want to involve in this project, he just need to execute the following command:
`make tools`
I don't prefer to introduce every tool here. BTW, from my point, this looks like a develop guide. So the chapter name might be 'Develop Environment'.